op-batcher/batcher/throttler/controller_test.go

```diff
@@ -466,6 +497,39 @@ func TestIntensityToParams(t *testing.T) {
 	}
 }
 
+// TestIntensityToParams_UnlimitedStrategy verifies that when using the unlimited strategy,
+// the controller ignores size limits and returns zeros regardless of intensity.
+func TestIntensityToParams_UnlimitedStrategy(t *testing.T) {
+	controller := NewThrottleController(NewUnlimitedStrategy(), testThrottleConfig)
+
+	tests := []struct {
+		name              string
+		intensity         float64
+		expectedIntensity float64
+	}{
+		{"negative intensity", -0.5, 0.0},
+		{"zero intensity", 0.0, 0.0},
+		{"half intensity", 0.5, 0.5},
+		{"max intensity", 1.0, 1.0},
+		{"above max intensity (clamped)", 1.5, 1.0},
```

Comment:
shouldn't the expected intensity always be 0? and since it should, couldn't you just remove it as a parameter from the test table?

Reply:
I realised why this is this way, now. This test targets the common code which translates intensity (output by the controller) to "params" (essentially the max tx and block size).

The way it is currently implemented on this PR, the unlimited controller _could_ output any intensity (although it will only ever output 0) and the translation code would just ignore it anyway. 

This does prompt me to think about the abstractions here though. Controllers should only express an intensity, but both the disabled and unlimited controllers are actually reaching outside of their abstractions and doing far more (the former disables _all_ throttling code, the latter disables the default/always-on block size throttling). So we are breaking / abusing the interfaces here.